src/cargo/util/command_prelude.rs

```diff
@@ -1174,6 +1178,122 @@ fn get_target_triples_from_rustc() -> CargoResult<Vec<clap_complete::CompletionC
         .collect())
 }
 
+pub fn get_package_candidates() -> Vec<clap_complete::CompletionCandidate> {
```

Comment:
we should clarify this is `get_pkg_id_spec_candidates` with this change.

I only realized this as I wondered if we could reuse this for `cargo info` (maybe) and `cargo add` (no)

Reply:
Change done! Thanks!